assets/js/base/components/quantity-selector/stories/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { boolean } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import QuantitySelector from '../';
+import './style.scss';
+
+export default {
+	title: 'WooCommerce Blocks/@base-components/QuantitySelector',
+	component: QuantitySelector,
+};
+
+export const Default = () => (
+	<QuantitySelector
+		disabled={ boolean( 'Disabled', false ) }
+		itemName='widgets'
+	></QuantitySelector>
```

Comment:
If it doesn't have children, it can be a self-closing component.
```suggestion
	/>
```